.github/workflows/private-clone.yml

```diff
@@ -0,0 +1,28 @@
+name: Clone Private Repo Test
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - dev-[0-9]+.[0-9]+.[0-9]+
+  push:
+    branches:
+      - develop
+    tags:
+      - v*
+  workflow_dispatch:
+
+jobs:
+  clone:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+      - name: Clone Private Repo
+        uses: actions/checkout@v4
+        with:
+          repository: NASA-AMMOS/aerie-orbiter-model
+          path: aerie-orbiter-model
+          ref: main
```

Comment:
this should be possible if you create a fine-grained token (with read access to `aerie-orbiter-model`) secret and reference it like:

```yaml
    token: ${{ secrets.READ_ONLY_TOKEN }}
```